assembl/static/widget/share/index.html

```diff
@@ -22,29 +19,261 @@
 	box-sizing: border-box;
 	width: 100%;
 	max-width: 1000px;
+	padding: 3px;
```

Comment:
Is it necessary to put css in the index? Is it possible to create a specific css file?